### ISSUE TITLE
Meeting: TIA Fiscal SharePoint Site consultation

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#6677]
### Objective 
Figure out what Nelson is wanting to do with TIA Fiscal

### Participants
Nelson E., Diana

### Agenda
Add agenda here or create agenda from [this template](https://docs.google.com/document/d/1d_49KW5C_vSz8Bs50v-cxyIJuTNJMwMrh7ypcxRHgZI/edit#) and add link. 

------
- [x] Schedule meeting
- [ ] Optional: Schedule debrief
- [x] Meet and take notes
- [ ] Optional: Debrief with DTS team members
- [ ] Create resulting issues

**Comment by dianamartin** [comment]
@amenity @TracyLinder  FYI

We talked about what Nelson was wanting to do, create a SharePoint site for his new program TIA Fiscal. 
He showed me there was a GIS web map for `Mitigation Database` and it was shared with him by Sangeeta. It seems like it was requested to be put on AGOL back in the day, owned by ATD Publisher. Not sure it it's even been maintained since. 

I also showed him the TDSD Portal>TIA Module. That has TIA Cases and Mitigations on it. He felt like that was overkill, but I had concern that he was asking for something that was going to duplicate data and have employees searching on a SharePoint site and a Knack app. 

I said perhaps we should have him put in a DTS ticket and we could scope out what he wants and we can determine the best solution. I was suggesting that we do a join on the Mitigations Database feature layer against his own TIA Fiscal spreadsheet and see which records need to be added (spatially). Then when the datasets are aggregated we can figure out how he can maintain it in GIS. 

I said GIS bc depending on what the feature looks like perhaps we could consume it into the TDSD/TIA map/table. This definitely needs scoping though and I'd like to get John's opinion on it.
